index.html

```diff
@@ -1500,13 +1507,13 @@ <h3>Selective Disclosure</h3>
       After validation, the Verifier will have the following data for further processing:
       </p>
       <p class="note">
-        A benefit of using <code>vc+ld+jwt</code> and <code>vc+ld+sd-jwt</code> 
+        A benefit of using <code>vc+json+jwt</code> and <code>vc+ld+json+sd-jwt</code> 
```

Comment:
```suggestion
        One benefit of using <code>vc+json+jwt</code> and <code>vc+ld+json+sd-jwt</code> 
```

Reply:
@TallTed  I edited your suggestion, because we don't actually have `vc+json+jwt` in this spec... and we probably will never see that, given IETF commentary on it.

Reply:
OH, mine was an accident. I did not intend `vc+json+jwt`; I did intend `vc+ld+json+jwt` (though I still think this should just be `jwt`, *maybe* with `;profile=application/vc+ld+json`, which I *think* would mean that the `typ` value must be `application/jwt;profile=application/vc+ld+json`, not `jwt;profile=application/vc+ld+json`).